dg/media/app_dashboards/js/loop_dashboard.js

```diff
@@ -2821,36 +2869,74 @@ function aggregator_payment_sheet(data_json, aggregator) {
             for (var j = 0; j < data_set.length; j++) {
                 if (data_set[j].indexOf(payments_gaddidar_contribution[i]['date']) != -1 && data_set[j].indexOf(payments_gaddidar_contribution[i]['mandi__name']) != -1) {
                     data_set[j][8] += parseFloat(payments_gaddidar_contribution[i]['amount']);
-                    data_set[j][9] = (data_set[j][9] - parseFloat(payments_gaddidar_contribution[i]['amount'])).toFixed(2);
+                    data_set[j][9] = parseFloat((data_set[j][9] - parseFloat(payments_gaddidar_contribution[i]['amount'])).toFixed(2));
                     break;
                 }
             }
         }
     }
 
+
+    
+
+    var formatVal = function (yourNumber) {
+    //Seperates the components of the number
+        var n= yourNumber.toString().split(".");
+    //Comma-fies the first part
+        if(n[0])
+            {
+                n[0] = n[0].replace(/\B(?=(\d{3})+(?!\d))/g, ","); //convert numbers of the form 1156 -> 1,156
+            }
+            if(n[1] != null)
+            {
+                var a = n[1].toString();
+                n[1] = parseInt(a.charAt(0));
+            }
+            return n.join(".");
+        }
+
+
+    var finalFormat = function (value){
+        if(value.indexOf('.') === -1)
+            return value + '.0';
+        else
+            return value;
+    }
+
+        // Remove the formatting to get integer data for summation
+    var intVal = function ( column_value ) {
```

Comment:
Please justify the use of intVal function, couldn't find any case where column_value is not 'number'